builtin/process_osh.py

```diff
@@ -682,4 +683,87 @@ def Run(self, cmd_val):
         return 0
 
 
+class Kill(vm._Builtin):
+    """Send a signal to a process"""
+    def __init__(self, job_control, job_list, waiter):
+        # type: (process.JobControl, process.JobList, Waiter) -> None
+        self.job_control = job_control
+        self.job_list = job_list
+        self.waiter = waiter
+        self.exec_opts = waiter.exec_opts
```

Comment:
not using exec_opts?

actually it seems like we're not using most of these args?    Please remove them if not